client/web/compose/src/views/Admin/Pages/Edit.vue

```diff
@@ -353,6 +353,8 @@
                       />
 
                       <c-input-confirm
+                        v-if="layout.pageLayoutID !== '0'"
```

Comment:
You should remove the ifs, since you can remove layouts that you just added aswell, same for layout actions below